lib/evaluateArithmetic.js

```diff
@@ -13,9 +13,15 @@ function evaluateArithmeticDFS(node) {
   let innerNodeStatus;
   if (NodeType.isParenthesis(node)) {
     innerNodeStatus = evaluateArithmeticDFS(node.content);
+    if (innerNodeStatus.hasChanged()) {
```

Comment:
Can we just call this status? that's what you call it in other places. Or should it always be innerNodeStatus?